megaqc/user/models.py

```diff
@@ -33,22 +44,22 @@ class User(db.Model, CRUDMixin):
 
     __tablename__ = 'users'
     user_id = Column(Integer, primary_key=True)
-    username = Column(String(80), unique=True, nullable=False)
-    email = Column(String(80), unique=True, nullable=False)
-    salt = Column(String(80), nullable=True)
-    password = Column(Binary(128), nullable=True)
+    username = Column(String(80, convert_unicode=True), unique=True, nullable=False)
```

Comment:
Does `convert_unicode=True` not need to be added to every `Column` for every table in this case?

Reply:
Ah, right right. I can't think of a reason why not. Could probably make a `UnicodeString` subclass that just passes everything to the `String` init, but also sets that flag so we don't need to worry about forgetting it somewhere, too.

Reply:
Cool - thanks for investigating! Are you happy to add this in for a future PR?

Reply:
Sure thing

Reply:
Ok, so bad news - this change has broken the login for Python 2 😆

Reply:
Instead of returning a string, this column now returns:

```
<read-only buffer for 0x7f80d76bfeb0, size 73, offset 0 at 0x7f80cf1fcab0>
```

This breaks all of the downstream password handling..

Reply:
Sorry, locally with sqlite it works for new users registered after the change. But postgres with py2 doesn't work with logins for even new users. I guess this is probably why the `password` field was previously set to `Binary` instead of `String`, though I have no idea why we're getting this.

Reply:
Hm. Okay, that's weird. I'll try and sort it out.

Reply:
No, I have no idea. And don't worry about it - if you can't replicate it then I'll just clobber our database and start again (or at least the users table). We don't have any real data in it yet.

Reply:
Yeah, easiest solution would be just to clobber the old users table, if that's an option.

Think I might try and get tests working then, so we can avoid some of these in the future.

Reply:
Deleted, recreated and all is working 👍